pkg/profiler/cpu/cpu.go

```diff
@@ -331,9 +409,19 @@ func (p *CPU) addUnwindTableForProcess(pid int) {
 }
 
 func (p *CPU) prefetchProcessInfo(ctx context.Context, pid int) {
-	if _, err := p.processInfoManager.Fetch(ctx, pid); err != nil {
+	procInfo, err := p.processInfoManager.Fetch(ctx, pid)
+	if err != nil {
 		level.Debug(p.logger).Log("msg", "failed to prefetch process info", "pid", pid, "err", err)
 	}
+
+	// TODO: This should only be called once.
+	if procInfo.Interpreter != nil {
+		err := p.bpfMaps.addInterpreter(pid, *procInfo.Interpreter)
+		if err != nil {
+			// Must never fail.
```

Comment:
Still, let's not panic and stop the whole profiler. There might be other processes we would be interested in the system.

Reply:
Agreed, this was a stray panic and we should not stop the world if this fails